styles/styles.css

```diff
@@ -973,6 +978,11 @@ main .block.centered > div {
     margin-bottom: 2rem;
   }
 
+  main .section.bottom-dark-border .section-container {
+    border-bottom: 2px solid var(--primary);
+    font-weight: var(--font-weight-bold);
```

Comment:
I don't think it's a bug, maybe it's about authoring. Seems the the words is surround by <strong> tag which doesn't require setting font-weight.
<img width="366" alt="image" src="https://github.com/hlxsites/sunstar-foundation/assets/57093988/b0240e19-5e07-41a1-9bce-5851a15354d6">

Reply:
yes, this was not needed. thanks for calling this out.